pkg/framework/plugins/nodeutilization/highnodeutilization_test.go

```diff
@@ -47,15 +49,15 @@ func TestHighNodeUtilization(t *testing.T) {
 
 	testCases := []struct {
 		name                string
-		thresholds          api.ResourceThresholds
+		thresholds          componentconfig.ResourceThresholds
```

Comment:
Let's keep `api.ResourceThresholds` in all cases for now

Reply:
reallly? but you and a7i said change to below. so do i need to use ` api.ResourceThresholds` in args or use `componentconfig.ResourceThresholds` in args
```
type LowNodeUtilizationArgs struct {
	metav1.TypeMeta

	UseDeviationThresholds bool
	Thresholds             ResourceThresholds
	TargetThresholds       ResourceThresholds
	NumberOfNodes          int
}
```

Reply:
```go
type LowNodeUtilizationArgs struct {
	metav1.TypeMeta

	UseDeviationThresholds bool
	Thresholds             api.ResourceThresholds
	TargetThresholds       api.ResourceThresholds
	NumberOfNodes          int
}
```
`api.ResourceThresholds` in all the places.

Reply:
get, thanks for clear that, i will update sooon